jenkinsbuilder/cccp_index_reader.py

```diff
@@ -24,8 +24,10 @@ def projectify(
     new_project[0]['project']['name'] = appid
     new_project[0]['project']['git_url'] = giturl
     new_project[0]['project']['git_branch'] = gitbranch
-    new_project[0]['project']['rel_path'] = ('/' + gitpath) \
-        if (gitpath and not gitpath.startswith('/')) else '/'
+    rel_path = "/"
+    if gitpath and gitpath != "/":
```

Comment:
Well this would assume that the user always passes a proper git path or ''. If a None is passed even by mistake, then:
```

In [1]: a=None

In [2]: rp = "/"

In [3]: a, rp
Out[3]: (None, '/')

In [4]: if a != "/":
   ...:     rp = a if a.startswith("/") else (rp + a)
   ...:     
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-fb79f32750b6> in <module>()
      1 if a != "/":
----> 2     rp = a if a.startswith("/") else (rp + a)
      3 

AttributeError: 'NoneType' object has no attribute 'startswith'
```